scipy/stats/tests/test_hypotests.py

```diff
@@ -4,10 +4,10 @@
 from numpy.testing import (assert_, assert_equal, assert_allclose,
                            assert_almost_equal)  # avoid new uses
 
+import pytest
 from pytest import raises as assert_raises
 from scipy.stats._hypotests import (epps_singleton_2samp, cramervonmises,
-                                    _cdf_cvm)
-import scipy.stats as stats
+                                    _cdf_cvm, barnard_exact)
```

Comment:
Oops, I'l fix this. I'm going to check locally whether i messed something else up.